bindings/pyroot_experimental/pythonizations/python/ROOT/_facade.py

```diff
@@ -179,3 +179,42 @@ def _setattr(self, name, val):
     @property
     def __version__(self):
         return self.gROOT.GetVersion()
+
+    # Overload VecOps namespace
+    @property
+    def VecOps(self):
+        if not 'VecOps' in self.__dict__:
+            ns = self._fallback_getattr('VecOps')
+            self.__dict__['VecOps'] = ns
+            try:
+                from libROOTPythonizations import AsRVec
+                ns.AsRVec = AsRVec
+            except:
+                pass
+        return self.__dict__['VecOps']
```

Comment:
Assuming we raise an error if we can't pythonize, here I would just `return ns`

Reply:
Agree, if we throw a hard error.